src/test/java/CucumberTestRunner.java

```diff
@@ -0,0 +1,17 @@
+package com.mariamkatamashvili.gym;
+
+import org.junit.platform.suite.api.ConfigurationParameter;
+import org.junit.platform.suite.api.IncludeEngines;
+import org.junit.platform.suite.api.SelectClasspathResource;
+import org.junit.platform.suite.api.Suite;
+
+import static io.cucumber.core.options.Constants.GLUE_PROPERTY_NAME;
+import static io.cucumber.core.options.Constants.PLUGIN_PROPERTY_NAME;
+
+@Suite
+@IncludeEngines("cucumber")
+@SelectClasspathResource("features")
+@ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "com.mariamkatamashvili.gym.messaging")
+@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty")
```

Comment:
Cucumber option 'tags' is not presented, so impossible to meet the following requirement:
Tests can be run from the command line, with the ability to select a particular set of tests, either for a single endpoint or for all.